.github/CONTRIBUTING.md

```diff
@@ -56,15 +56,29 @@ Adhering to the following process is the best way to get your work included in t
    git pull [--rebase] upstream master
    ```
 
-5. Squash your commits down to a single one (we want to keep the master branch nice and clean)
+5. Set-up the website
```

Comment:
Please add some language indicating that you really don't need to run the website to propose a new question.  Honestly, I imagine that people just create PRs right on GitHub.com, most of the time. This is good information for people who want to run or test the site, however.